connector/src/main/java/org.apache.flink/connector/nebula/table/NebulaDynamicTableFactory.java

```diff
@@ -160,6 +179,7 @@ private NebulaClientOptions getClientOptions(ReadableConfig config) {
                 .setGraphAddress(config.get(GRAPHADDRESS))
                 .setUsername(config.get(USERNAME))
                 .setPassword(config.get(PASSWORD))
+                .setTimeout(config.get(TIMEOUT))
```

Comment:
Previously we were not able to configure timeout for the Table API connector because this line was missing. I believe this is a bug.

Reply:
good catch!